### ISSUE TITLE
Use components without auto importing?

### ISSUE THREAD

**Issue by 684efs3** [nuxt/content#840]
Is it possible to use components inside markdown with the auto import component feature of Nuxt turned off? I've found that with auto import set to false, the hot reload is a lot faster. 

I turned auto import off, and in my _slug.vue (where I am calling the content files), I import the components I use in my markdown, but they don't show up.

**Comment by NozomuIkuta** [comment]
I guess you are seeing warning messages like `[Vue warn]: Unknown custom element: - did you register the component correctly?`.

Without auto-importing, you have to register the components manually by yourself, according to the scope of usage (global or local registration). Otherwise, Nuxt (or actually Vue) can't resolve them.

**Comment by 684efs3** [comment]
So I import the components locally like below, but this doesn't work. Is there something I am missing?

```
<script>
import ComponentA from "../../components/ComponentA.vue"

export default {
components: {
ComponentA
}
}

</script>
```

**Comment by ManasMadrecha** [comment]
Yes, it sometimes doesn't work. I tried several restarting options like restarting my `npm run dev`, reopening folder in new window in Visual Studio Code, even restarting the PC... and then, it works.

In one of my projects, neither worked, so I deleted the local folder and re-downloaded it from GitHub and then it worked.

I think there's some bug in Nuxt because it doesn't update if one changes the location of the component from `global` folder to normal components. Then, you have to do the above steps.
